react-infinite/build.boot

```diff
@@ -30,7 +31,7 @@
    (sift :include #{#"^cljsjs"})
 
    (deps-cljs :name "cljsjs.react-infinite"
```

Comment:
When using requires without `cljsjs.` prefix, the name should also skip the prefix.